internal/saucecloud/cloud.go

```diff
@@ -393,10 +384,10 @@ func (r *CloudRunner) runJobs(jobOpts chan job.StartOptions, results chan<- resu
 }
 
 // remoteArchiveProject archives the contents of the folder to a remote storage.
-func (r *CloudRunner) remoteArchiveProject(project interface{}, folder string, sauceignoreFile string, dryRun bool) ([]string, error) {
+func (r *CloudRunner) remoteArchiveProject(project interface{}, folder string, sauceignoreFile string, dryRun bool) (string, []string, error) {
```

Comment:
I suppose the existing method was already not well documented 😅 , but now would be good to do so. It's not clear from the go doc or the return signature what the string and the slice represent.

Naming the return params would help make the code more self-documenting 🙏